azext_hanaonazure/_params.py

```diff
@@ -11,4 +11,11 @@
 def load_arguments(self, _):
     with self.argument_context('hanainstance') as c:
         c.argument('resource_group_name', arg_type=resource_group_name_type)
-        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
\ No newline at end of file
+        c.argument('instance_name', options_list=['--instance-name', '-n'], help="The name of the SAP HANA instance", id_part='name')
```

Comment:
See above; it is not clear what is meant by "name of the SAP HANA instance": The SID? The resource name? An arbitrary name the user can assign to a SAP HANA instance?

Reply:
I don't actually make any change to this line. This is there before. The only reason that it marks the line as changed is because I added a newline after it to add the monitoring parameters